app/assets/templates/widget_time_options_template.html

```diff
@@ -26,7 +26,14 @@
             <i class="icon-fb"></i>
           </button>
         </span>
-        <input type="text" class="form-control" datetime-picker datetime="endTime" data-format="yyyy-MM-dd" placeholder="Until">
+        <span class="until_container">
+          <input type="text" class="form-control" datetime-picker datetime="endTime" data-format="yyyy-MM-dd" placeholder="Until">
+          <span class="until_clear"
+            ng-click="clearEndTime()"
+            tooltip-append-to-body="true" tooltip="Clear">
+            <i class="glyphicon glyphicon-remove-circle"></i>
```

Comment:
Bonus points if the remove icon only appears when the end time is not already set to `null`.